apps/src/aichat/views/chatWarningModal.module.scss

```diff
@@ -0,0 +1,70 @@
+@import "color", "font";
+
+.chatWarningModal {
+  hr {
+    margin: 1rem 0;
+    border-color: $light_gray_200;
+  }
+
+  a {
+    font-family: $gotham-bold;
+    font-size: 1rem;
+    color: $brand_secondary_default;
+    margin: 1rem 0;
+    text-decoration: underline;
+    transition: color ease-in-out .2s;
+
+    &:hover {
+      color: $brand_secondary_dark;
+    }
+  }
+}
+
+.xCloseButton {
+  // Clear default button styles
+  background: none;
+  color: inherit;
+  border: none;
+  padding: 0;
+  font: inherit;
+  cursor: pointer;
+  outline: inherit;
+  box-shadow: none;
+
+  position: absolute;
+  top: 1rem;
+  right: 1rem;
+
+  i {
+    color: $light_gray_300;
+    font-size: 1.1875rem;
+    line-height: 1.4844rem;
+    width: 1.4844rem;
+  }
+
+  &:hover {
+    // Clear default button styles
+    box-shadow: none;
+    background: none;
+    color: inherit;
+    border: none;
+    padding: 0;
+    font: inherit;
+    cursor: pointer;
+    outline: inherit;
+
+    i {
+      color: $light_gray_500;
+    }
+  }
+
+  &:focus {
+    outline: $brand_primary_default solid 2px;
+    outline-offset: 2px;
+  }
```

Comment:
I am curious about the outline around the x button. When I click away from the dialog, and then click back on the dialog, the outline doesn't seem to appear. Is this expected behavior?

Reply:
I wonder if a student is on their phone if we need this to be scrollable. Are we assuming that a user is on a laptop and not a mobile device?

Reply:
I think we will want students to be able to scroll up in the chat to see previous chats if needed, but we dont need an autoscroll feature. Assume students are using a laptop for MVP.

Reply:
RE Outline: works as expected.
RE Modal scrolling on mobile: I'm using `AccessibleDialog` component here developed by @code-dot-org/acquisition-products-team (?) assuming it already keeps in mind the mobile users. From what I see in styles and chrome devTools - it should render correctly in default phone sizes; can not check on the phone locally, need this to be merged to public so I can test from phone. If mobile will be an issue - we might fix it by adding custom modal className/styles